workloads/network-perf-v2/README.md

```
@@ -14,47 +14,69 @@ Running from CLI:
-|TCP RR| 1 | 64 - 1024 - 8192 | 2 | 30 |
+| Test | Parallelism | MessageSize | Samples | Duration |
+|------|-------------|-------------|---------|----------|
+| TCP Stream | 1, 2, 4 | 64, 1024, 8192 | 2 | 30 |
+| TCP RR | 1 | 64, 1024, 8192 | 2 | 30 |
+
+### MicroShift
+
+Set `PLATFORM=microshift` to run against a single-node MicroShift cluster:
+
+```sh
+$ PLATFORM=microshift WORKLOAD=full-run.yaml ./run.sh
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)